examples/demo/src/dashboard/Dashboard.tsx

```diff
@@ -73,9 +67,9 @@ const Dashboard: FC = () => {
             }
         );
         const aggregations = recentOrders
-            .filter(order => order.status !== 'cancelled')
+            .filter((order: Order) => order.status !== 'cancelled')
```

Comment:
This shouldn't be necessary as `recentOrders` is already an `Order[]` thanks to `getList`